uber_analytics/site_sections/map.py

```diff
@@ -56,6 +56,40 @@ def radial_zip_data(self, out, session, **params):
                                       VincentyDistance((x["Latitude"], x["Longitude"]), center_coord).miles,
                                       "%.2f" % float(self.zips_counter[x['Zipcode']]/total_count * 100)])
 
+    @csv_file
+    def attendee_in_radius_csv(self, out, session, radius, **params):
```

Comment:
hey, only thing that's confusing here is, this function will filter attendees which don't have .can_spam set (which, for the email data we're trying to collect, is exactly the right behavior), but, this function being named ```attendee_in_radius_csv``` might lead someone to believe that it will spit out all attendees, which it won't.

I'd suggest we either 
1) rename this to ```attendee_in_radius_ok_to_email``` (easiest thing)
2) keep the name but let it pass in a filter param here and the caller can specify a filter of ```can_spam=True```